internal/pkg/utils/reflect.go

```diff
@@ -14,7 +14,7 @@ type StructField struct {
 }
 
 func (f *StructField) JsonName() string {
-	if f.jsonName != "" {
+	if f.jsonName != "" && f.jsonName != "-" {
```

Comment:
Pouziva sa ako nazov polozky v `diff-e`, ak ma field `json` name, tak sa pouzije to, ... no `Relations` ma `json:"-"`, tj. do JSONu sa neuklada, ... v tom pripade tiez nepouzijeme JSON name, ale nazov fieldu zo struktury.